tests/testAutopas/tests/cells/CellOwnershipTest.cpp

```diff
@@ -6,81 +6,102 @@
 
 #include "CellOwnershipTest.h"
 
+/**
+ * The CellOwnershipTests are only executed for FullParticleCell, ReferenceParticleCell, SortedCellView and
+ * OctreeLeafNode. Note: For OctreeNodeWrapper (which also inherits from ParticleCell) these tests are not executed,
+ * since it only manages internal nodes that store the particles. Only the OctreeLeafNodes store the particles in an
+ * internal storage.
+ * The ClusterTowers (which also inherits from ParticleCell) internally use a FullParticleCell to store particles.
+ */
+
 using CellOwnershipTestingTypes =
     ::testing::Types<autopas::FullParticleCell<Molecule>, autopas::ReferenceParticleCell<Molecule>>;
```

Comment:
This doesn't really fit together. The comment suggests that the tests are executed for `FullParticleCell`, `ReferenceParticleCell`, `SortedCellView` and `OctreeLeafNode`. However, the tuple only contains `FullParticleCell` and `ReferenceParticleCell`.

Reply:
Okay, I adjusted the tests a bit. A general comment explains why the tests are not executed for ClusterTower and OctreeNodeWrapper. 
The Typed Test is integrated into all other tests (including SortedCellView and OctreeLeafNode) because it makes little sense to have this test for only two different types of cells.